Rubberduck.Core/App.cs

```diff
@@ -231,11 +232,13 @@ public void LogRubberduckStart()
         {
             var version = _version.CurrentVersion;
             GlobalDiagnosticsContext.Set("RubberduckVersion", version.ToString());
+            var osProductName = Registry.GetValue(@"HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows NT\CurrentVersion", "ProductName", "").ToString();
```

Comment:
We probably want the registry reads in a try/catch block, and fall back to `Environment.OSVersion.VersionString` if something goes wrong, no?

Reply:
Sure!

Reply:
Next commit makes it pass empty strings, which would look like:

    Operating System:     (Microsoft Windows NT 10.0.19044.0)

Large blank spot where the friendly name would be.